apps/music/views/playlists/view.js

```diff
@@ -49,34 +49,32 @@ var PlaylistsView = View.extend(function PlaylistsView() {
 
   this.client.on('databaseChange', () => this.update());
 
-  this.update();
+  this.update()
+    .then(() => this.list.scrollTo(this.searchBox.HEIGHT));
```

Comment:
I only want to `scrollTo()` when the view *first* renders, not every time the list content updates.